stream.html

```diff
@@ -22,7 +22,7 @@
 <div id="overlay-top">
 	<div id="overlay-top-content" class="sbox-small flair hidden">
 		<div class="content">
-			<p><span class="match-type">League:</span><span class="match-id">??/??</span></p>
+            <p><span class="match-id">Match 4</span> <span class="match-type">(league)</span></p>
```

Comment:
Any reason to bake in the match number etc? Leaving placeholders means its obvious there's no data rather than it being incorrect.

```suggestion
            <p><span class="match-id">Match ??</span> <span class="match-type">(??)</span></p>
```